src/scripts/main.js

```diff
@@ -2,6 +2,24 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const body = document.querySelector('body');
+  const block = document.createElement('div');
+  const hTwo = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  hTwo.className = 'title';
+  hTwo.innerHTML = `${title}`;
```

Comment:
```suggestion
  hTwo.innerText = title;
```
Ти передаєш сюди значення змінної. Немає сенсу використовувати інтерполяцію